phpstan.neon.dist

```diff
@@ -0,0 +1,5 @@
+includes:
+  - ./vendor/openeuropa/code-review-drupal/dist/phpstan.drupal.neon
+parameters:
+  excludePaths:
+  	- modules/oe_content_timeline_field/src/TimelineFieldProcessor.php
```

Comment:
It probably makes sense to add a comment to indicate the reason for excluding some files from the scanning.